.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        test-commands: ["npm run hospital"]
+        test-commands: ["npm run demo"]
```

Comment:
While all the refactor that we discussed we can consider that as part of separate story but can you fix the strategy.matrix? The test-commands doesnt sounds right and we dont need a strategy anymore to begin with... We just need a `target_env` - either through workflow dispatch or default `demo` and then later just

```
- name: Run test
        run: xvfb-run npm run ${{ env.target_env }}
```
you would then fix the reports name because the current report names have npm command into it 😀

```
with:
          name: bahmni-e2e-reports(${{ env.target_env }})-${{ github.sha }}
          path: ./reports
```

Reply:
updated the file-

- Removed Strategy.matrix 
- added workflow dispatch and made demo as default env for on push & daily schedule triggers.
- updated the report name as `name: bahmni-e2e-reports(${{env.TARGET_ENV}})-${{env.TEST_CONTEXT}}` , easy to identify against which tags & env the test has been executed.

let me know if changes looks good!